src/main/java/uk/gov/hmcts/darts/event/model/DarNotifyEvent.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.darts.event.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.extern.jackson.Jacksonized;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@Data
+@Builder
+@Jacksonized
+public class DarNotifyEvent {
```

Comment:
name possibly ok, but want to make sure we know what its for. Would `NotifyDarPcRequest` be more descriptive?

Reply:
:) I settled on this mainly because of https://github.com/hmcts/darts-legacy/blob/2e512caf650aeb532fbdad539a714cd191a7144c/workspace/dartsdev/darts-webservice/src/main/java/com/VIQSoultions/www/DARNotifyEventClass.java